### PR TITLE
#161254620 view/edit profile

### PR DESCRIPTION
**Pull request by abadojack**
- user can view their profile
- user can edit their profile

[Delivers #161254620]

#### What does this PR do?
``` Allow user to edit/view their profile```
#### Description of Task to be completed?
```
- user can view their profile
- user can edit their profile
 ```
#### How should this be manually tested?
```
- [view profile](https://ah-orcas-frontend-staging.herokuapp.com/profile) 
- edit the profile using the edit button
 ```
#### What are the relevant pivotal tracker stories?
 [#161254620](https://www.pivotaltracker.com/story/show/161254620)

#### Screenshots (if appropriate)
screenshot 2018-12-07 at 09 45 28

#### Checklist:

- [X] My code follows the style guidelines of this project
- [ ] At least 2 people have reviewed my PR
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] My PR has one commit.

### CODE REVIEW

**Comment by Eyansky** [comment]
You still have hound issues.

**Comment by abadojack** [comment]
> You still have hound issues.

@Eyansky I have fixed all the hound violations

**Comment by Eyansky** [comment]
Looks good to me lets wait on codeclimate and coveralls to complete.

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  return-statements:
+    config:
+      threshold: 8
+  similar-code:
+    config:
+      threshold: 150
+  identical-code:
+    config:
+      threshold: 250
+
```
Comment (code-sleuth) [inline .codeclimate.yml]: Add a new line at the end of this file

File: src/utils/user.util.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export default () => {
+  try {
+    return JSON.parse(localStorage.getItem('user'));
+  } catch (e) {
+    return null;
```
Comment (code-sleuth) [inline src/utils/user.util.js:5]: why are you suppressing errors here?

**Review by code-sleuth — CHANGES_REQUESTED** [review]

File: src/utils/user.util.js (hunk 2 above)
Comment (abadojack) [reply src/utils/user.util.js:5]: It's easier to validate since we expect a valid `user` or `null`.

**Comment by code-sleuth** [comment]
we still have PR format issues at this point.

**Review by code-sleuth — APPROVED** [review]
LGTM

**Pull request merged**